bahmni-reports/package/helm/templates/secrets.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Chart.Name }}-secrets
+type: Opaque
+data:
+  OPENMRS_DB_HOST: {{ .Values.secrets.OPENMRS_DB_HOST | b64enc | quote }}
+  OPENMRS_DB_USERNAME: {{ .Values.secrets.OPENMRS_DB_USERNAME | b64enc | quote }}
+  OPENMRS_DB_PASSWORD: {{ .Values.secrets.OPENMRS_DB_PASSWORD | b64enc | quote }}
+  REPORTS_DB_SERVER: {{ .Values.secrets.REPORTS_DB_SERVER | b64enc | quote }}
+  REPORTS_DB_USERNAME: {{ .Values.secrets.REPORTS_DB_USERNAME | b64enc | quote }}
+  REPORTS_DB_PASSWORD: {{ .Values.secrets.REPORTS_DB_PASSWORD | b64enc | quote }}
+  OPENELIS_DB_SERVER: {{ .Values.secrets.OPENELIS_DB_SERVER | b64enc | quote }}
+  ODOO_DB_SERVER: {{ .Values.secrets.ODOO_DB_SERVER | b64enc | quote }}
+  ODOO_DB_USERNAME: {{ .Values.secrets.ODOO_DB_USERNAME | b64enc | quote }}
+  ODOO_DB_PASSWORD: {{ .Values.secrets.ODOO_DB_PASSWORD | b64enc | quote }}
```

Comment:
We won't be running odoo, can we get rid of these env variables

Reply:
removed these variables